### ISSUE TITLE
`Your Brave wallet is powered by Uphold` is not translated

### ISSUE THREAD

**Issue by btlechowski** [brave/brave-browser#12012]
This is a regression in 1.5.71, Not reproducible in 1.5.70.

## Steps to Reproduce

   1. Run Brave in Polish locale
   2. Enable rewards
   3. Open `brave://rewards`

## Actual result:

`Your Brave wallet is powered by Uphold` is not translated
![image](https://user-images.githubusercontent.com/34715963/95198863-32b73880-07dc-11eb-8bbd-cfe9206fc070.png)

## Expected result:
`Your Brave wallet is powered by Uphold` is translated
![image](https://user-images.githubusercontent.com/34715963/95198905-45ca0880-07dc-11eb-8133-5f61d2a804b8.png)

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

Brave | 1.15.71 Chromium: 86.0.4240.72 (Official Build) (64-bit)
-- | --
Revision | 581582174c512f44f44fd1aea340471f54b2365f-refs/branch-heads/4240@{#1134}
OS | Ubuntu 18.04 LTS

cc @brave/legacy_qa @mkarolin @rebron

**Comment by LaurenWags** [comment]
Reproduced on macOS using French and German with

```

Brave | 1.15.71 Chromium: 86.0.4240.72 (Offizieller Build) (x86_64)
-- | --
Überarbeitung | 581582174c512f44f44fd1aea340471f54b2365f-refs/branch-heads/4240@{#1134}
Betriebssystem | macOS Version 10.14.6 (Build 18G3020)

```

**Comment by GeetaSarvadnya** [comment]
Reproduced in `Windows 10 x64 - 1.15.71`

**Comment by kjozwiak** [comment]
Closing and moved into https://github.com/brave/brave-browser/milestone/135 as https://github.com/brave/brave-core/pull/6788 has been merged into `1.15.x` 👍

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.15.72 Chromium: 86.0.4240.75 (Offizieller Build) (x86_64)
-- | --
Überarbeitung | c69c33933bfc72a159aceb4aeca939eb0087416c-refs/branch-heads/4240@{#1149}
Betriebssystem | macOS Version 10.14.6 (Build 18G3020)
```

Verified STR from description for both French and German:

**`French`**
<img width="418" alt="12012 French" src="https://user-images.githubusercontent.com/28145373/95338816-52546c80-0881-11eb-97d4-74d3cde0b222.png">


**`German`**
<img width="397" alt="12012 German" src="https://user-images.githubusercontent.com/28145373/95338821-554f5d00-0881-11eb-8a05-c71e2653e1ed.png">

----------
